src/Microsoft.ML.TorchSharp/NasBert/TextClassificationTrainer.cs

```diff
@@ -346,13 +350,29 @@ public Trainer(TextClassificationTrainer parent, IChannel ch)
                 LearningRateScheduler = torch.optim.lr_scheduler.OneCycleLR(
                    Optimizer.Optimizer,
                    max_lr: _parent._options.LearningRate[0],
-                   total_steps: _parent._options.MaxUpdate,
+                   total_steps: ((rowCount / _parent._options.BatchSize) + 1) * _parent._options.MaxEpoch,
```

Comment:
>BatchSize [](http://example.com/codeflow?start=62&length=9)

I know this is unlikely but is it possible someone can set the batch size to 0?

Reply:
Yeah its possible. Let me add some validation that the parameters provided fall within the expected range.

Reply:
:thumbsup:

Reply:
Validation added when the estimator is created.